src/NServiceBus.Persistence.ServiceFabric/SagaPersister/ServiceFabricSagaAttribute.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace NServiceBus.Persistence.ServiceFabric
+{
+    using System;
+
+    [AttributeUsage(AttributeTargets.Class, Inherited = false)]
+    public sealed class ServiceFabricSagaAttribute : Attribute
```

Comment:
Since this is a public API, it should provide xmldoc on the class and it's public members.